arduino/libraries/RFM69_LowPowerLab/Examples/DeepSleep_usingLowPowerLibrary/DeepSleep_usingLowPowerLibrary.ino

```diff
@@ -39,25 +39,25 @@ void setup () {
     flash.sleep();
 #endif
 
-  for (uint8_t i=0; i<=A5; i++)
```

Comment:
There's an old trick in C that you can comment out a large section with #if 0/#endif

Reply:
?